benchmarks/dynamo/common.py

```diff
@@ -1355,6 +1356,16 @@ def load(cls, model, example_inputs, device):
                 # see https://github.com/pytorch/pytorch/issues/113029
                 example_outputs = copy.deepcopy(model)(*example_args, **example_kwargs)
 
+            def _produce_dynamic_shapes(path, x):
+                # mark_dynamic() is ignored for export.
+                # use this to produce dynamic_shapes spec instead.
+                if not isinstance(x, torch.Tensor):
+                    return None
+                return {
```

Comment:
You sure you don't have to handle other markers?

Reply:
In these tests I'm pretty sure it's only mark_dynamic that's used..